<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo12-addon-repair-quality-control-issue</title>
  </head>
  <body>
    <h1>Links for odoo12-addon-repair-quality-control-issue</h1>
    <a href="https://files.pythonhosted.org/packages/1b/81/65f8688fc99783d75834226c7b5bd47d0e12d27c36e985cf6b7539e1b420/odoo12_addon_repair_quality_control_issue-12.0.1.0.0.99.dev3-py3-none-any.whl#sha256=83451c2e6c785e3e15623b13d8edc6150cfdb9061a86bd3b1e5f0eb1a92bddf3" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.0.99.dev3-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/c2/8e/6d770c59bf003911d92ee67bdd8f6c69bc4386a35a0c62451e6aafb394d2/odoo12_addon_repair_quality_control_issue-12.0.1.0.1-py3-none-any.whl#sha256=8d4e9c3be33ca526bb8c5a85d8cfe1ecf38543babe6ed0c0390184f75f038966" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.1-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/7e/4f/9ea9b362659c438467d39ced3bf82d1fba7a1a49e1697ab991ac11583ee0/odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev2-py3-none-any.whl#sha256=76f27d6f5cd32261a77b018bb215409b66e643e96603662123564149dc08a6aa" data-requires-python="&gt;=3.5" >odoo12_addon_repair_quality_control_issue-12.0.1.0.1.99.dev2-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 16453255-->